discovery-provider/src/tasks/index_rewards_manager_backfill.py

```diff
@@ -581,35 +580,56 @@ def process_solana_rewards_manager(
 
 
 def check_if_backfilling_complete(
-    session: Session, solana_client_manager: SolanaClientManager
+    session: Session, solana_client_manager: SolanaClientManager, redis: Redis
 ) -> bool:
-    stop_sig_tuple = (
-        session.query(IndexingCheckpoint.signature)
-        .filter(
-            IndexingCheckpoint.tablename == index_rewards_manager_backfill_tablename
+    try:
+        redis_complete = redis.get(index_rewards_manager_backfill_complete)
+        if redis_complete:
+            redis_complete = str(redis_complete.decode())
+        if redis_complete == "true":
+            return True
```

Comment:
nit: since we're just checking for the existance of this key in redis, we don't need this comparison

Reply:
ooohhh I see.